delphi-checks/src/main/java/au/com/integradev/delphi/checks/PlatformDependentCastCheck.java

```diff
@@ -54,7 +56,18 @@ public DelphiCheckContext visit(ArgumentListNode argumentList, DelphiCheckContex
         Type castedType = getHardCastedType(argumentList, context);
```

Comment:
Oops, I must have forgotten to migrate PlatformDependentCast to use CastUtils. Would it be possible for you to do this as part of this change? I can also do it in a separate PR if preferred.

Reply:
Separate PR preferred on this one.